### PR TITLE
Add docs for applications endpoint

### PR DESCRIPTION
**Pull request by Fredar**

### CODE REVIEW

File: content/reference/changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2017-05-29
+
+  * [doc update] Add docs for `applications` endpoint.
```
Comment (StoneFrog) [inline content/reference/changelog.md]: Api improvement? I think it's new feature, not just update of docs.

Comment (Fredar) [inline content/reference/changelog.md]: Its all mixed up in the changelog tbh.
Latest addition of endpoint is labeled as `doc update`

**Comment by Fredar** [comment]
Requires update with proper scope for this endpoint first

**Comment by Fredar** [comment]
@StoneFrog updated, please recheck

**Comment by Fredar** [comment]
@StoneFrog updated :smile:

**Pull request merged**